usr/src/tools/scripts/nightly.sh

```diff
@@ -196,11 +197,14 @@ function build {
 
 	echo "\n==== Build errors ($LABEL) ====\n" >> $mail_msg_file
 	egrep ":" $SRC/${INSTALLOG}.out |
-		egrep -e "(^${MAKE}:|[ 	]error[: 	\n])" | \
-		egrep -v "Ignoring unknown host" | \
-		egrep -v "cc .* -o error " | \
-		egrep -v "warning" | tee $TMPDIR/build_errs${SUFFIX} \
-		>> $mail_msg_file
+	    egrep -e "(^${MAKE}:|[ 	]error[: 	\n])" | \
+	    egrep -v "Ignoring unknown host" | \
+	    egrep -v "cc .* -o error " | \
+	    egrep -v "warning" | tee $TMPDIR/build_errs${SUFFIX} \
+	    >> $mail_msg_file
+	cat $SRC/${INSTALLOG}.out | \
```

Comment:
Unnecessary cat.
```
sed -n "/^Undefined[ 	]*first referenced$/,/^ld: fatal:/p" \
     < $SRC/${INSTALLOG}.out >> $mail_msg_file
```

Reply:
Thanks Dan. I've pulled in your suggestion.